sklearn/utils/multiclass.py

```diff
@@ -227,7 +200,7 @@ def is_multilabel(y):
     True
 
     """
-    return is_label_indicator_matrix(y) or is_sequence_of_sequences(y)
```

Comment:
effectively we should be renaming `is_label_indicator_matrix` (not the other way, which is my reading of your suggestion, @rvraghav93)